README.md

```diff
@@ -577,3 +579,25 @@ $ stack build --test --no-run-tests
 $ cd install
 $ stack build
 ```
+
+### Hacking on haskell-language-server
+
+#### Introduction tutorial
+Pepeiborra [wrote an tutorial](https://github.com/pepeiborra/hls-tutorial) on writing a plugin in HLS.
+
+#### Test your hacked HLS in your editor
+If you want to test HLS while hacking on it, follow the steps below.
+
+To do once:
+- Open some codebase on which you want to test your hacked HLS in your favorite editor
+- Configure this editor to use your custom HLS executable
+  - With Cabal: `<path to HLS project>/haskell-language-server/dist-newstyle/build/<os>/<ghc version>/haskell-language-server-<HLS version>/x/haskell-language-server/build/haskell-language-server/haskell-language-server`
```

Comment:
A better way to get this path is `cabal exec which haskell-language-server

Reply:
I have added this command, though I also kept the full path as I'm not sure all editors support a command as a path

Reply:
Friendly weekly 😝  reminder that `which`does not work in windows:
```
PS D:\dev\ws\haskell\hls> cabal exec which haskell-language-server
Resolving dependencies...
cabal.exe: The program 'which' is required but it could not be found.

PS D:\dev\ws\haskell\hls> cabal exec where haskell-language-server
D:\dev\ws\haskell\hls\dist-newstyle\build\x86_64-windows\ghc-8.8.4\haskell-language-server-0.3.0.0\x\haskell-language-server\build\haskell-language-server\haskell-language-server.exe
D:\bin\haskell-language-server.exe
```

Reply:
So... Should I remove those 2 commands, and only leave the path?

Reply:
I think you should remove the path since that's an unstable part of cabal, and mention both `which` and `where` commands.

Reply:
Done!

Reply:
Thanks!